### PR TITLE
fix: allow absolute paths for the wgc compose output file

### PR DESCRIPTION
**Pull request by fiam**
The join is not necessary for relative paths and breaks absolute paths

## Motivation and Context

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: cli/src/commands/router/commands/compose.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default (opts: BaseCommandOptions) => {
     });
 
     if (options.out) {
-      await writeFile(join(process.cwd(), options.out), routerConfig.toJsonString());
```
Comment (StarpTech) [inline cli/src/commands/router/commands/compose.ts:102]: Is it backwards compatible? Wouldn't `resolve()` solve it too?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: cli/src/commands/router/commands/compose.ts (hunk 1 above)
Comment (fiam) [reply cli/src/commands/router/commands/compose.ts:102]: Arguably, `pathe.join()` should be smart enough to yield the absolute path when `process.out` is absolute but that's not the case.

On the other hand, there's no need to prepend the current directory to `options.out`, a relative path is already resolved relative to the current working directory. Joining `cwd()` with it manually is just incorrect.

Comment (fiam) [reply cli/src/commands/router/commands/compose.ts:102]: Forgot to answer your actual question 🙈. The change is backwards compatible, it only affects absolute paths which are not working correctly right now (they're interpreted as relative to the current working directory)

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**